charts/swoop-bundle/templates/swoop-bundle-db-migration-job.yaml

```diff
@@ -11,12 +11,27 @@ spec:
       name: "{{ .Values.dbMigration.jobName }}-{{ .Values.dbMigration.version }}"
     spec:
       containers:
-      - name: db-migration-job # Replace with DB migration job
-        image: gcr.io/k8s-staging-perf-tests/sleep:latest
-        args: ["15s"]
-        resources:
-          requests:
-            cpu: 1
-            memory: "200Mi"
+      - name: db-migration-job
+        image:  quay.io/element84/swoop-db # replace with test cont
+        command: ["python", "/opt/swoop/db/scripts/run-migration-job.py"]
+        env:
+        - name: PGHOST
+          value: "{{.Values.postgres.service.name}}.{{.Values.dbMigration.namespace}}.svc.cluster.local"
```

Comment:
```suggestion
          value: "{{.Values.postgres.service.name}}.{{.Values.dbMigration.namespace}}"
```

Reply:
I think the reachable host would be, for example: `postgres.db` and not `posgres.db.svc.cluster.local`